src/site/facilities/facilities_health_services_buttons.drupal.liquid

```diff
@@ -3,12 +3,12 @@
 {% endcomment %}
 <div data-template="facilities/facilities_health_services_buttons" class="vads-l-row">
     <div class="medium-screen:vads-l-col--4 vads-l-col--12 vads-u-margin-right--2p5">
-        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/make-an-appointment" onClick="recordEvent({ event: 'cta-primary-button-click' });">Make an appointment</a>
+        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/make-an-appointment">Make an appointment</a>
     </div>
     <div class="medium-screen:vads-l-col--4 vads-l-col--12 vads-u-margin-right--2p5">
-        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/register-for-care" onClick="recordEvent({ event: 'cta-primary-button-click' });">Register for care</a>
+        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/register-for-care">Register for care</a>
     </div>
     <div class="medium-screen:vads-l-col--4 vads-l-col--12 vads-u-text-align--right vads-u-flex--fill">
-        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-margin-x--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/pharmacy" onClick="recordEvent({ event: 'cta-primary-button-click' });">Pharmacy</a>
+        <a class="usa-button vads-u-margin-top--1 vads-u-margin--0 vads-u-margin-x--0 vads-u-width--full vads-u-font-size--md" href="/{{ path }}/pharmacy">Pharmacy</a>
```

Comment:
Per this https://github.com/department-of-veterans-affairs/va.gov-team/issues/7345#issuecomment-636945687 , this should have been the event for the default button, not primary